plugins/modules/zos_copy.py

```diff
@@ -114,7 +114,7 @@
         be deleted and recreated following the process outlined in the C(volume) option.
       - When the C(dest) is an existing VSAM (RRDS), then the source must be an RRDS.
         The VSAM (RRDS) will be deleted and recreated following the process outlined
-        in the C(volume) option.
+       in the C(volume) option.
```

Comment:
This is outdented 1 - the 'i' should be under the 'T'

Reply:
Good catch, thank you.